subsys/bluetooth/controller/ll_sw/ull_conn_types.h

```diff
@@ -196,15 +196,6 @@ struct ll_conn {
 			uint32_t ticks_to_offset;
 		} periph;
 #endif /* CONFIG_BT_PERIPHERAL */
-
-#if defined(CONFIG_BT_CENTRAL)
-		struct {
-#if defined(CONFIG_BT_CTLR_CONN_META)
-			uint8_t  is_must_expire:1;
```

Comment:
@mtpr-ot can this be removed?

Reply:
I think we would need to remove the full "must-expire" feature first, as there are dependencies in ULL. It's used for calculating the latency.

Reply:
I don't know to what extent the feature is used by other uses/vendors, but we're not using it anymore. However, the MUST_EXPIRE feature ties into the current ticker implementation, so it may be a larger overhaul to safely remove.